lib/Conversion/PTensorToLinalg/PTensorToLinalg.cpp

```diff
@@ -293,19 +295,63 @@ static BodyType buildTrivial(::mlir::Type typ) {
   };
 }
 
+// Builder for TOSA body.
+// The builder function takes an extra agrument ::mlir::Type
+// Many TOSA functions take two arguments and do the
+// same operation on both arguments.
```

Comment:
Is this the functionality we need? Why is TOSA any better than math and arith?

Reply:
@fschlimb I found those logical ops only in TOSA, is there any other alternative?

Reply:
I don't think so. An alternative is of course to write them manually.